### ISSUE TITLE
Completion gets distorted when output is larger than window size

### ISSUE THREAD

**Issue by TLINDEN** [fi-ts/cloudctl#188]
In a proper sized terminal window the output of (e.g.) `cloudctl ip <tab>` looks like this:
```
% c ip 
allocate  (allocate a static IP address for your project that can be used for your cluster's service type load balancer)
delete    (delete an ip)
list      (list ips)
static    (make an ephemeral ip static such that it won't be deleted if not used anymore)
```

However, if the window does not fit the text lines without breaking anymore (that is, if one increases font size or decreases the window size) then the output of `cloudctl ip <tab>` looks like this:
```
% c ip 
$allocate  (allocate a static IP address for your project that can be used for your cluster\s
service
type342200246) delete\ \ \ \ \(delete\ an\ ip\) list\ \ \ \ \ \ \(list\ ips\) static\ \ \ \ \(make\ an\ ephemeral\ ip\ static\ such\ that\ it\ won\t be deleted if not used anymore)
```

The behavior is the same with all completions which contain help texts.

So it seems `cloudctl` is aware of the window size and tries to adapt somehow, but this fails. I'd suggest not to adapt at all.

Env:
```
bash 5.1.4(1)-release

% c version 
client: v0.10.19 (72d20d71), tags/v0.10.19-0-g72d20d7, 2022-06-21T07:45:10Z, go1.18.3
server:
    builddate: "2022-06-20T13:47:20Z"
    gitsha1: 4897315c
    name: cloud-api
    revision: tags/v0.18.23-0-g4897315
    version: v0.18.23
```

**Comment by majst01** [comment]
Pretty sure this depends on your terminal implementation, cannot reproduce with neither gnome-shell nor console (the successor of the former)

![image](https://user-images.githubusercontent.com/410110/176124400-73f0a0d1-301e-43e2-a3f2-ff052910f442.png)

**Comment by TLINDEN** [comment]
My default console is xfce-term and I can reproduce it in konsole (the kde one) and xterm.

**Comment by majst01** [comment]
code completion is not implemented by us, we simply use the completion framework provided by the [cobra](https://github.com/spf13/cobra) cli framework, which is used by almost all go based cli tools, including `kubectl`, `helm` and others.

**Comment by TLINDEN** [comment]
Ok so you think I shall open an issue at the cobra project?

**Comment by majst01** [comment]
First you could try completion debugging as described here: https://github.com/spf13/cobra/blob/master/shell_completions.md#debugging

**Comment by TLINDEN** [comment]
All right, now it's getting weird.

I tried as suggested, but the debugging output was identical to the expected (not actual) output.

Then - in order to submit an issue - I created a minimal completion app based on the example in the docs. Running this example's completion in a too small window worked fine, no distortions.

But then I tried `cloudctl` and - believe it or not - now it worked here as well!

And even today a couple of days later after updating cloudctl, the issue seems to be fixed. Other than installing the golang environment nothing changed on my system. However, the issue persists with other tools like `kubectl`.

While I don't really understand whats going on here, from my point of view it's fixed and the issue could be closed anyway.
